recipes/dye-score/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "dye-score" %}
+{% set version = "0.1.0" %}
+{% set hash_type = "sha256" %}
+{% set hash = "0c50f781b5f65845840dfe86b69c601f81ae723ea63aed57d32e8ec4994b62e88" %}
+
+package:
+  name: {{ name }}
```

Looking at this again, I think this could be a [noarch](http://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-python) package:
```suggestion
  noarch: python
  name: {{ name }}
```